images/win/scripts/Installers/Install-NodeLts.ps1

```diff
@@ -11,7 +11,16 @@ New-Item -Path $PrefixPath -Force -ItemType Directory
 New-Item -Path $CachePath -Force -ItemType Directory
 
 $defaultVersion = (Get-ToolsetContent).node.default
-Choco-Install -PackageName nodejs -ArgumentList "--version=$defaultVersion"
+$ODataQuery = "`$filter=(Id eq `'nodejs`') and (IsPrerelease eq false) and (Version ge `'$defaultVersion`')"
```

Comment:
use the same name conversion, all variables should be in lowercase.

Reply:
You don't need to escape single quotes.

Reply:
Also, maybe use more clear name, for example `queryFilter`